lib/win32/windows/time_calc_helper.rb

```diff
@@ -0,0 +1,120 @@
+module Windows
+  module TimeCalcHelper
+
+    # Returns actual no of days for given month;
+    # Array with a 0 is defined to give actual result without
+    # any manipulation. eg, DAY_OF_MONTH[1] = 31
+    # 0(NUMBER) is kept to avoid exceptions during calculations
+    DAY_OF_MONTH = [0, 31, 28, 31, 30, 31, 30, 31, 31, 30, 31, 30, 31]
+
+    # Returns no of days in a given month of a year
+    def day_in_month(month, year)
```

Comment:
And this to get_days_in_a_month for clarity?

Reply:
As per ruby best practices, we should generally avoid using 'get_' and 'set_' in method names. This is a trivial technique we often use in Java/C/C++.(Got this idea while working with Rubocop). Few reasons are listed [here](https://stackoverflow.com/questions/26097084/why-does-rubocop-or-the-ruby-style-guide-prefer-not-to-use-get-or-set)
Still, If it is needed for better understanding, will update the code accordingly.
Kindly let us know your views about this
Thank you